File: Selnium script for testing/Script with Python/test_scenario_6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_6():
    """
    Dummy test run for scenario 6
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_6")

    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_225")
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_980")
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_206")
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.ID, "checkout_btn").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_788")
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_6()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario6.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario6 {
    public static void executeTest6() {
        /*
         * Dummy test run for scenario 6
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_6");

        try {
            Thread.sleep(2000);
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.className("nav-item")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.cssSelector(".promo-code")).click();
            driver.findElement(By.id("user_name")).click();
            driver.findElement(By.cssSelector(".promo-code")).sendKeys("test_data_400");
            driver.findElement(By.className("nav-item")).sendKeys("test_data_594");
            driver.findElement(By.id("checkout_btn")).click();
            driver.findElement(By.cssSelector(".promo-code")).click();
            driver.findElement(By.xpath("//button[@type='submit']")).click();
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_972");
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest6();
    }
}
